tv_apps/tv-ftu/style/index.css

```diff
@@ -0,0 +1,86 @@
+html {
+  font-size: 10px;
+  /* We have Fira Sans as default font in real device. But it isn't default in
+   * browser. That's the main reason this line is here.*/
+  font-family: "Fira Sans";
+  font-weight: 100;
+  font-style: italic;
+}
+
+html, body {
+  width: 100%;
+  height: 100%;
+  overflow: hidden;
+  margin: 0;
+  padding: 0;
+  background: #000;
+}
+
+body {
+  background: url('images/background.png') no-repeat;
+}
+
+.txt-mid {
+  font-size: 3.8rem;
+}
+
+.txt-large {
+  font-size: 7rem;
+}
+
+.txt-center {
+  text-align: center;
+}
+
+smart-button[type=rounded-corner-text] {
+  width: 23.7rem;
+  font-size: 2.4rem;
+  font-weight: normal;
+}
+
+.slide {
+  width: 100%;
+  height: 100%;
+  color: #fff;
+  position: fixed;
+}
+
+.slide-section {
+  position: absolute;
+  margin-right: -50%;
+  left: 50%;
+  transform: translate(-50%, 0px);
+}
+
+.slide-section.top {
+  top: 7.5rem;
+}
+
+.slide-section.mid {
```

Comment:
And since Evelyn is taking care of running the app in simulator (which is running in a smaller screen), I would suggest you fix the problem of running in smaller screen if possible. (Or you can just discuss with Evelyn)